modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -2381,7 +2401,7 @@
               "application/json": {
                 "example": {
                   "data": {
-                    "type": "intentToFileValidation",
+                    "type": "intent_to_file",
```

Comment:
should we include `_validation` here? I feel like we should since it would match the original, just need to make sure we're aligning snakecase.